### PR TITLE
Call endpoint to mark upload status as completed

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

**Comment by suzannehamilton** [comment]
@techncl @MancunianSam Could we maybe have a quick call today to tidy up the git history for this branch? There are WIP commits, commits with duplicate names, and some of Rooha's commits that have already been merged, so I'm not sure what's going on.

I'm happy to do to the git wrangling if you tell me what the changes should be!

**Comment by techncl** [comment]
> @techncl @MancunianSam Could we maybe have a quick call today to tidy up the git history for this branch? There are WIP commits, commits with duplicate names, and some of Rooha's commits that have already been merged, so I'm not sure what's going on.
> 
> I'm happy to do to the git wrangling if you tell me what the changes should be!

Sure. It's weird, I rebased the "WIP" commit and then this happened.

File: npm/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "./dist/",
     "noImplicitAny": true,
-    "module": "es6",
+    "module": "esnext",
```
Comment (suzannehamilton) [inline npm/tsconfig.json:5]: @MancunianSam What are the implications of this? Will it cause any problems?

**Review by suzannehamilton — COMMENTED** [review]
A few question for Sam. I'm wondering if the right thing to do is to downgrade webpack on tdr-generated-graphql until we can fix these issues properly, rather than leaving this with a lot of changes that we don't understand?

**Review by suzannehamilton — DISMISSED** [review]
🎉

**Review by suzannehamilton — APPROVED** [review]

**Pull request merged**